subsys/bluetooth/host/Kconfig

```diff
@@ -48,17 +48,19 @@ config BT_RX_BUF_LEN
 	  (4 bytes) and the ACL header (also 4 bytes) which yields 73 bytes.
 
 config BT_HCI_TX_STACK_SIZE
+	# NOTE: By _not_ having a prompt here, Kconfig will fall back on
+	# the default value if none of the other values are set.
```

Comment:
Could say something like "Since this symbol isn't user-configurable (has no prompt), it will always get its value from the defaults."

There's a lot of those comments strewn randomly in the Kconfig files though. I wonder how much they help in practice. Maybe it would be better to remove all of them and add more documentation on how prompts work somewhere else.

The [Kconfig - Tips and Best Practices](https://docs.zephyrproject.org/latest/tools/kconfig/index.html) page has some handy tips btw.

Reply:
I've read all the manuals I could come across :+1: I generally agree that duplication of information like this is a bad thing. In this case, the note is there to warn anyone that making this a prompt will break the automagic selection of stack size depending on controller selected etc. That's the _real_ problem that we're trying to solve, but I don't know any method that can actually safe-guard against this without putting a lot of burden on the user :/

Reply:
Maybe this would work, if you think there's a risk someone might add a prompt later: "This value is derived from other symbols and should not be user-configurable. Do not give it a prompt."